include/picongpu/simulation/stage/MomentumBackup.hpp

```diff
@@ -52,17 +53,14 @@ namespace stage
                 VectorAllSpecies,
                 momentumPrev1
             >::type;
-            pmacc::meta::ForEach<
-                SpeciesWithMomentumPrev1,
-                particles::Manipulate<
-                    particles::manipulators::unary::CopyAttribute<
-                        momentumPrev1,
-                        momentum
-                    >,
-                    bmpl::_1
-                >
-            > copyMomentumPrev1;
-            copyMomentumPrev1( step );
+            using Manipulator = particles::manipulators::unary::CopyAttribute<
```

Comment:
Could you please give the manipulator another name e.g updatePrevMom

Reply:
Thanks, fixed and rebased.